content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -175,7 +175,7 @@ The following plugins support `WaitForFirstConsumer` with dynamic provisioning:
 
 The following plugins support `WaitForFirstConsumer` with pre-created PersistentVolume binding:
 
-- All of the above
+- [GCEPersistentDisk](#gce-pd)
```

Comment:
I was wondering if the other storage-classes like such as aws ebs support the `WaitForFirstConsumer` attriubute.

The yaml manifest described here does mention the `WaitForFirstConsumer`:

https://aws.amazon.com/blogs/storage/simplifying-amazon-ebs-volume-migration-and-modification-using-the-ebs-csi-driver/

Reply:
I think we should add **`aws ebs`** to the `WaitForFirstConsumer` list.

Reply:
> I think we should add **`aws ebs`** to the `WaitForFirstConsumer` list.

Does Kubernetes have support for AWS EBS storage in-tree? https://kubernetes.io/docs/concepts/storage/volumes/#awselasticblockstore suggests not.

Reply:
No, aws ebs is removed from v1.27